endorsement/dao/shared_drive.py

```diff
@@ -467,13 +467,19 @@ def managers_for_shared_drive(google_drive_states):
             # TODO: wrap below in a transaction?
 
             # update drive manager list
-            managers = managers_for_shared_drive(drive_states)
-            shared_drive.members.set(managers)
-
-            # update drive name
-            if shared_drive.drive_name != drive_state.drive_name:
-                shared_drive.drive_name = drive_state.drive_name
-                shared_drive.save()
```

Comment:
Oof, sorry for the betrayal here.

Stats on the current report shows 129 as the largest drive name length. I love off-by-1...